File: solutions/codechef/PROGLANG/PROGLANG_3.cpp
// DATE: 20/08/2022, 01:10:13
// PROBLEM NAME: Programming Languages
// PROBLEM URL: https://www.codechef.com/problems/PROGLANG
// PROBLEM DIFFICULTY RATTING: 1001
// STATUS: accepted
// TIME: 0.00
// MEMORY: 5.1M

#include <iostream>
using namespace std;

int main() {
    int t;
    cin>>t;
    while(t--){
        int a,a1,a2,b,b1,b2;
        cin>>a>>b>>a1>>b1>>a2>>b2;
        if((a==a1 || a==b1) && (b==a1 || b==b1))
            cout<<1<<endl;
        else if((a==a2 || a==b2) && (b==a2 || b==b2))
            cout<<2<<endl;
        else{
            cout<<0<<endl;
        }
    }
    
	return 0;
}

File: solutions/codechef/PROGLANG/PROGLANG_3.py
# DATE: 26/09/2022, 08:04:55
# PROBLEM NAME: Programming Languages
# PROBLEM URL: https://www.codechef.com/problems/PROGLANG
# PROBLEM DIFFICULTY RATTING: 1001
# STATUS: accepted
# TIME: 0.02
# MEMORY: 9.5M

for _ in range(int(input())):
    a,b,a1,b1,a2,b2 = map(int, input().split(" "))
    if (a==a1 or a==b1) and (b==a1 or b==b1):
        print(1)
    elif (a==a2 or a==b2) and (b==a2 or b==b2):
        print(2)
    else:
        print(0)

